### PR TITLE
Update changelog and notifier version

### PR DESCRIPTION
**Pull request by chimanjain**

### CODE REVIEW

**Comment by chimanjain** [comment]
PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 ## master
 
-## [v5.6.0][v5.6.0] (July 22, 2022)
+## [v5.6.1][v5.6.1] (December 14, 2022)
+
+* Dependency updates
```
Comment (kyrylo) [inline CHANGELOG.md:7]: Either don't mention this or mention what dependencies were updated.

Comment (chimanjain) [reply CHANGELOG.md:7]: Removing dependencies as we don't need to mention that.

**Comment by chimanjain** [comment]
PTAL

**Review by kyrylo — APPROVED** [review]

**Pull request merged**